sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -399,6 +399,7 @@ queryOrganization
       (SORT BY sort+=sortItem (',' sort+=sortItem)*)?
       windowClause?
       (LIMIT (ALL | limit=expression))?
+      (OFFSET offset=expression)?
```

Comment:
does SQL standard define the order of LIMIT and OFFSET clauses? Here we force users to write LIMIT clause first.

Reply:
Yes, SQL standard defined the order for `LIMIT` and `OFFSET` clauses. `LIMIT` at the front of `OFFSET`.
The behavior looks contrary to MySQL direction. 
`MySQL` support `LIMIT 10, OFFSET 10`.